lesson-02/task-03.js

```diff
@@ -0,0 +1,15 @@
+const findLongestWord = function (sentense) {
+  let longestWord;
+  let num = 0;
+  let array = sentense.trim().split(" ");
+  for (let i = 0; i < array.length; i++) {
+    if (array[i].length > num) {
```

Comment:
Зайва змінна не потрібна, так як можна перевірити довжину найдовшого слова
```suggestion
    if (array[i].length > longestWord.length) {
      longestWord = array[i];
      }
```